src/App.js

```diff
@@ -104,23 +106,28 @@ function App () {
             </h1>
           </Link>
         </header>
-        <Switch>
-          <Route path='/miners/:minerId/deadlines/:deadlineId'>
-            <Deadline client={client} miners={miners} head={head} />
-          </Route>
-          <Route path='/miners/:minerId'>
-            <MinerInfo client={client} miners={miners} head={head} />
-          </Route>
-          <Route path='/full'>
-            <Full client={client} miners={miners} />
-          </Route>
-          <Route path='/status'>
-            <Status head={head} spa={spa} client={client} miners={miners} />
-          </Route>
-          <Route path='/'>
-            <Home miners={miners} />
-          </Route>
-        </Switch>
+        <div className="container">
```

Comment:
we should remove this, each View should have it's own container by default I think (?) if not, then ok

Reply:
The whole switch statement was *not* inside a container, only the top bar, the logo that's why i added one.